x/incentives/types/msgs.go

```diff
@@ -5,16 +5,22 @@ import (
 	"time"
 
 	lockuptypes "github.com/osmosis-labs/osmosis/v10/x/lockup/types"
+	txfeestypes "github.com/osmosis-labs/osmosis/v10/x/txfees/types"
 
 	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
 const (
 	TypeMsgCreateGauge = "create_gauge"
 	TypeMsgAddToGauge  = "add_to_gauge"
+
+	createGaugeMinBaseFee = 2
+	addToGaugeMinBaseFee  = 1
```

Comment:
Looking for suggestions on the values here please

@ValarDragon @czarcas7ic

Reply:
We can do 50 OSMO for create and 25 for add for now I think